compositor_render/src/renderer/color_converter_pipeline.rs

```diff
@@ -0,0 +1,236 @@
+use super::{
+    common_pipeline::{RectangleRenderBuffers, Sampler, U32Uniform, Vertex},
+    texture::{RGBATexture, YUVTextures},
+    WgpuCtx,
+};
+
+const PRIMITIVE_STATE: wgpu::PrimitiveState = wgpu::PrimitiveState {
+    polygon_mode: wgpu::PolygonMode::Fill,
+    topology: wgpu::PrimitiveTopology::TriangleList,
+    front_face: wgpu::FrontFace::Ccw,
+    cull_mode: Some(wgpu::Face::Back),
+    strip_index_format: None,
+    conservative: false,
+    unclipped_depth: false,
+};
+
+pub struct YUVToRGBAConverter {
+    pipeline: wgpu::RenderPipeline,
+    sampler: Sampler,
+    buffers: RectangleRenderBuffers,
+}
+
+impl YUVToRGBAConverter {
+    pub fn new(
+        device: &wgpu::Device,
+        yuv_textures_bind_group_layout: &wgpu::BindGroupLayout,
+    ) -> Self {
+        let shader_module = device.create_shader_module(wgpu::include_wgsl!("yuv_to_rgba.wgsl"));
+        let sampler = Sampler::new(device);
+        let buffers = RectangleRenderBuffers::new(device);
+
+        let pipeline_layout = device.create_pipeline_layout(&wgpu::PipelineLayoutDescriptor {
+            label: Some("YUV to RGBA colour converter render pipeline layout"),
```

Comment:
```suggestion
            label: Some("YUV to RGBA color converter render pipeline layout"),
```
Let's get rid of the British spelling that elixir used to enforce on us